Modules.md

```diff
@@ -11,6 +16,13 @@ is called a **module**. A module contains:
 Sections declare their type and byte-length. Sections with unknown types are
 silently ignored.
 
+An instance contains:
+* a [linear memory](AstSemantics.md#linear-memory);
+* [global variable](AstSemantics.md#global-variables) state;
+* (when [threading](PostMVP.md#threads) is added) thread-local state;
```

Comment:
That sounds good to me. I indeed filed the above issue with an eye towards wasm, without full understanding of how asm.js implemented TLS. It looks like we agree on what wasm would do (including w.r.t. dynamic linking + TLS), so that's good.

I do think it's worth clarifying this point though: call TLS out explicitly. Is there anything else worth mentioning? We will have some form of shadow stack that can be inspected programatically (it will become per-thread eventually). There's also some implicit context that's provided by the embedder.